<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd"><html><head></head><body>

























































<div class="container-fluid main-container">




<div>




</div>


<div class="section level2">
<h2>NGS Analysis Spring 2023 Course Project (Option A)</h2>
<p>Differential gene expression (DGE) analysis with RNA-seq is an
essential method in many areas of biology including experimental
biomedicine. For your project, you will conduct a DGE analysis comparing
gene expression in control breast cancer cell lines to treatment lines
with a gene (NRDE2) that has been silenced with RNAi. The goal is to
characterize differentially expressed genes that may be impacted by
knocking down NRDE2.</p>
<p>A synopsis of the experiment can be found here:</p>
<p><a rel="noopener" href="https://www.ebi.ac.uk/ena/data/view/PRJNA490376?" class="uri">https://www.ebi.ac.uk/ena/data/view/PRJNA490376?</a></p>
<div class="section level3">
<h3>A note on independent completion of the project</h3>
<p>This assignment should be completed by you and is intended to reflect
your work. The assignment should not be completed in collaboration with
other students in the course or anyone else.</p>
</div>
<div class="section level3">
<h3>About the data</h3>
<p>The data include RNA-seq libraries from three biological replicates
from control cell lines and three biological replicates in which the
gene NRDE2 has been silenced with RNAi. The libraries were single-end
(SE) sequenced on an Illumina NextSeq platform.</p>
<p>To complete your project, you will need to process the raw data and
conduct DGE with the Salmon + tximport + DESeq2 workflow introduced in
the course in Week 10.</p>
<p>Specifically, you should do the following:</p>
<p>Trim the fastqs with fastp using appropriate settings to
automatically remove adapters from single end reads and polyG sequences
introduced on NextSeq platforms (see Week 2) The following is a table on
Greene with the samples and their fastq files:</p>
<p><code>/scratch/work/courses/BI7653/project.2023/project_fastqs.txt</code></p>
<p>The fastq files are located in:</p>
<p><code>/scratch/work/courses/BI7653/project.2023/fastqs</code></p>
<p>Run fastqc on the processed RNA-seq reads separately on each sample
and generate a MultiQC report.</p>
<p>Download the human reference transcriptome</p>
<p>Navigate to the GRCh38 genome “Gene Annotation” section of the
Ensembl website:</p>
<p><a rel="noopener" href="https://www.ensembl.org/Homo_sapiens/Info/Index" class="uri">https://www.ensembl.org/Homo_sapiens/Info/Index</a></p>
<p>Then click on “Download Fasta”-&gt;“cdna/” and download the file
“Homo_sapiens.GRCh38.cdna.all.fa.gz”</p>
<p>note: if you would like to download directly to hpc, you can copy the
link address and use wget.</p>
<p>g-unzip the transcripts fasta and run Picard tools NormalizeFasta to
strip everything after transcript id, which is the first identifier in
the Fasta header lines for each transcript.</p>
<p>Create Salmon index.</p>
<p><a rel="noopener" href="https://salmon.readthedocs.io/en/latest/salmon.html" class="uri">https://salmon.readthedocs.io/en/latest/salmon.html</a></p>
<p>Run Salmon in mapping-based mode using a command appropriate for
single-end data.</p>
<p><a rel="noopener" href="https://salmon.readthedocs.io/en/latest/salmon.html" class="uri">https://salmon.readthedocs.io/en/latest/salmon.html</a></p>
<p>Convert Salmon TPMs to gene-level counts with tximport and conduct
DGE with DESeq2.</p>
<p>tximport requires a mapping file that maps transcript ids to genes
which must be converted to imported as a data.frame (“tx2gene” in the
DESeq2 vignette). This file is provided to you as a comma-delimited
file:</p>
<p><code>/scratch/work/courses/BI7653/project.2023/tx2gene.csv</code></p>
</div>
<div class="section level3">
<h3>Completing your assignment</h3>
<p>Your final report should include a short introduction (1 paragraph
about DGE and the cancer cell line dataset), a materials and methods
section, a results section, and a brief discussion.</p>
<p>Items (a-c), and (f) below should be provided as a gzipped tar file
(i.e., with tar.gz file extension)</p>
<ol style="list-style-type: lower-alpha;">
<li>Provide all scripts and/or command lines you executed in completing
your assignment<br/>
</li>
<li>Provide the MultiQC report in html format<br/>
</li>
<li>Provide the Salmon output file with TPMs for each of the 6
samples<br/>
</li>
<li>Write a detailed methods section. This should include how you
trimmed the reads, anything unusual in the multiqc report, the library
type (stranded or unstranded?, can be inferred by Salmon), a description
of the workflow, whether you are reporting “shrunken” log fold-change
estimates (or not), the statistical approach used including the
multiple-test correction method.<br/>
</li>
<li>Include in your results section a table with the total number of
reads and the mapping rate for each sample, the number of statistically
significant genes at your chosen FDR, the number of biologically
relevant differentially expressed genes (defined using some criterion,
such as a change in gene expression of two-fold or greater), a table
with the 10 most highly significant differentially expressed genes, a
sample PCA, an MA plot, and dispersion-by-mean plot, and a raw p-value
histogram. Please also include any relevant comments about the
figures.<br/>
</li>
<li>Include the DESeq2 results for ALL Genes as a tab-delimited
file.<br/>
</li>
<li>Your report should be clear and concise with unambiguos descriptions
of results and methods.</li>
</ol>
<div class="section level4">
<h4>You are finished, upload your report and the *.tar.gz file and
include your name in both files. You can upload your files at the
“Assignments” section on the NYU Brightspace webpage.</h4>
</div>
</div>
</div>




</div>















<script type="module" src="https://s.brightspace.com/lib/bsi/20.23.5-231/unbundled/mathjax.js"></script><script type="text/javascript">document.addEventListener('DOMContentLoaded', function() {
					if (document.querySelector('math') || /\$\$|\\\(|\\\[|\\begin{|\\ref{|\\eqref{/.test(document.body.innerHTML)) {
						document.querySelectorAll('mspace[linebreak="newline"]').forEach(elm => {
							elm.setAttribute('style', 'display: block; height: 0.5rem;');
						});

						window.D2L.MathJax.loadMathJax({
							'outputScale': 1.3,
							'renderLatex': true
						});
					}
				});</script><script type="module" src="https://s.brightspace.com/lib/bsi/20.23.5-231/unbundled/prism.js"></script><script type="text/javascript">document.addEventListener('DOMContentLoaded', function() {
					document.querySelectorAll('.d2l-code').forEach(code => {
						window.D2L.Prism.formatCodeElement(code);
					});
				});</script></body></html>